crates/cxx-qt-gen/src/generator/rust/property/mod.rs

```diff
@@ -174,9 +157,7 @@ mod tests {
                         if self.rust().trivial_property == value {
```

Comment:
we could implement Deref quite easily which allows for `.rust()` to go here. DerefMut gets confused with the Pin stuff....

Reply:
`Deref` fun is here #586